api/api_storage.go

```diff
@@ -145,6 +145,8 @@ type StorageMiner interface {
 	// SealingSchedDiag dumps internal sealing scheduler state
 	SealingSchedDiag(ctx context.Context, doSched bool) (interface{}, error) //perm:admin
 	SealingAbort(ctx context.Context, call storiface.CallID) error           //perm:admin
+	//SealingSchedRemove removes a request from sealing pipeline
+	SealingRemoveRequest(ctx context.Context, SchedId uuid.UUID) error //perm:admin
```

Comment:
by convention param names usually start lower-case

```suggestion
	SealingRemoveRequest(ctx context.Context, schedId uuid.UUID) error //perm:admin
```